tests/e2e/testsuites/testsuites.go

```diff
@@ -0,0 +1,178 @@
+}
+
+func (t *StorageClassTest) Cleanup(client clientset.Interface, namespace *v1.Namespace) {
+	framework.Logf("deleting PersistentVolumeClaim %q/%q", namespace.Name, t.persistentVolumeClaim.Name)
+	err := client.CoreV1().PersistentVolumeClaims(namespace.Name).Delete(t.persistentVolumeClaim.Name, nil)
+	framework.ExpectNoError(err)
+	// Wait for the PV to get deleted if reclaim policy is Delete. (If it's
+	// Retain, there's no use waiting because the PV won't be auto-deleted and
+	// it's expected for the caller to do it.) Technically, the first few delete
+	// attempts may fail, as the volume is still attached to a node because
+	// kubelet is slowly cleaning up the previous pod, however it should succeed
+	// in a couple of minutes.
+	if t.persistentVolume.Spec.PersistentVolumeReclaimPolicy == v1.PersistentVolumeReclaimDelete {
+		By(fmt.Sprintf("deleting the claim's PV %q", t.persistentVolume.Name))
+		err = framework.WaitForPersistentVolumeDeleted(client, t.persistentVolume.Name, 5*time.Second, 10*time.Minute)
+		framework.ExpectNoError(err)
+	}
+	framework.Logf("deleting storage class %s", t.storageClass.Name)
+	err = client.StorageV1().StorageClasses().Delete(t.storageClass.Name, nil)
+	framework.ExpectNoError(err)
+}
+
+func NewStorageClassTestPod(command string) *StorageClassTestPod {
+	return &StorageClassTestPod{
+		pod: &v1.Pod{
+			TypeMeta: metav1.TypeMeta{
+				Kind:       "Pod",
+				APIVersion: "v1",
+			},
+			ObjectMeta: metav1.ObjectMeta{
+				GenerateName: "pvc-volume-tester-",
+			},
+			Spec: v1.PodSpec{
+				Containers: []v1.Container{
+					{
+						Name:         "volume-tester",
+						Image:        imageutils.GetE2EImage(imageutils.BusyBox),
+						Command:      []string{"/bin/sh"},
+						Args:         []string{"-c", command},
+						VolumeMounts: make([]v1.VolumeMount, 0),
+					},
+				},
+				RestartPolicy: v1.RestartPolicyNever,
+				Volumes:       make([]v1.Volume, 0),
+			},
+		},
+	}
+
+}
+
+func (p *StorageClassTestPod) SetupVolume(t *StorageClassTest, name, mountPath string) {
+	volumeMount := v1.VolumeMount{
+		Name:      name,
+		MountPath: mountPath,
+	}
+	p.pod.Spec.Containers[0].VolumeMounts = append(p.pod.Spec.Containers[0].VolumeMounts, volumeMount)
+
+	volume := v1.Volume{
+		Name: name,
+		VolumeSource: v1.VolumeSource{
+			PersistentVolumeClaim: &v1.PersistentVolumeClaimVolumeSource{
+				ClaimName: t.persistentVolumeClaim.Name,
+				ReadOnly:  false,
+			},
+		},
+	}
+	p.pod.Spec.Volumes = append(p.pod.Spec.Volumes, volume)
+}
+
+func (p *StorageClassTestPod) Provision(client clientset.Interface, namespace *v1.Namespace) {
+	var err error
+
+	p.pod, err = client.CoreV1().Pods(namespace.Name).Create(p.pod)
+	framework.ExpectNoError(err)
+
+	framework.WaitForPodSuccessInNamespaceSlow(client, p.pod.Name, namespace.Name)
+}
+
+func (p *StorageClassTestPod) Cleanup(client clientset.Interface, namespace *v1.Namespace) {
+	framework.Logf("deleting Pod %q/%q", p.pod.Name, namespace.Name)
```

Comment:
Nit: This will render as

```
deleting pod "name"/"namespace"
```

Please note that not only the order of the coordinates is "wrong", there are extra quotes.